docs/data/material/customization/theme-components/theme-components.md

```diff
@@ -77,17 +90,62 @@ const finalTheme = createTheme({
 
 {{"demo": "GlobalThemeOverrideCallback.js"}}
 
-### Using `sx` (experimental) syntax
+### Using the `sx` experimental syntax
+
+If you are not familiar with the `sx` prop, make sure you first check out [the concept around it](/system/the-sx-prop/) and [the difference with the `styled` utility](/system/styled/#difference-with-the-sx-prop).
+
+To have a similar shorthand CSS notation to the `sx` prop but within the theme, you can use `sx` inside the `stylesOverrides`.
```

Comment:
Rephrasing.

```suggestion
You can use the `sx` prop inside the `styleOverrides` key to modify styles within the theme using shorthand CSS notation.
```

Reply:
Can we elaborate on why a developer would want to do this? And this would also be a good place to explain why this is "experimental."

Reply:
@mnajdova anything you'd like to share about that?

Reply:
> Can we elaborate on why a developer would want to do this? 

Mainly for conistency. If a developer is already used to the `sx` syntax and it's shorthands, it's limiting that it can be used on only one place (the `sx` prop). More over, if they want to convert from `sx` prop to `styled()`, or move the styles to the theme overrides, they would have to manually change the styles to comply with the other regular syntax. This utility helps them by doing it automatically and providing a better DX along the way. 

It's experimental as it was added after the v5 stable release, and we usually export API like this as an experimental first, to see if there will be some unforeseen bugs and stabilize them later.

Reply:
Let me know what you think, tackled these aspects.